### PR TITLE
TCP reachability implementation

### PR DESCRIPTION
**Pull request by dkocher**

### CODE REVIEW

File: core/src/test/java/ch/cyberduck/core/diagnostics/HttpReachabilityTest.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+public class HttpReachabilityTest {
+
+    @Test
+    public void testIsReachable() {
+        final Reachability r = new HttpReachability();
+        assertTrue(r.isReachable(
+                new Host(new TestProtocol(Scheme.http), "test.cyberduck.io")
+        ));
+        assertTrue(r.isReachable(
```
Comment (ylangisc) [inline core/src/test/java/ch/cyberduck/core/diagnostics/HttpReachabilityTest.java:42]: Same test as in the line before.

File: core/src/main/csharp/ch/cyberduck/core/diagnostics/TcpReachability.cs
Comment (ylangisc) [inline core/src/main/csharp/ch/cyberduck/core/diagnostics/TcpReachability.cs:1]: File can be deleted?

File: core/dylib/src/test/java/ch/cyberduck/core/diagnostics/SystemConfigurationReachabilityTest.java
Comment (ylangisc) [inline core/dylib/src/test/java/ch/cyberduck/core/diagnostics/SystemConfigurationReachabilityTest.java:1]: What's the reason for the ignored tests?

Comment (dkocher) [reply core/dylib/src/test/java/ch/cyberduck/core/diagnostics/SystemConfigurationReachabilityTest.java:1]: Still registered and required for NetworkChangeMonitor.

**Review by ylangisc — CHANGES_REQUESTED** [review]
See comments.

**Review by ylangisc — APPROVED** [review]

**Pull request merged**